console/src/main/java/org/georchestra/console/ws/backoffice/users/UsersExport.java

```diff
@@ -53,6 +56,10 @@
 
     private UserInfoExporter accountInfoExporter;
 
+    @VisibleForTesting
+    @Value("${gdpr.enable}")
+    String gdprEnable;
```

Comment:
the VisibleForTesting annotation might suggest more than "package-private" modifier (i.e. public or so). I'd have put the member private, and defined getters/setters on it, but no idea how spring can react to this, to "autowire" it using the value annotation.

Also isn't spring capable of automatically casting true / false to a boolean ?

Reply:
relying on the rest of your PR, spring does not care about private member variables when it deals with the @Value annotation, as it probably does some Java reflection to access the field it needs to set.